GUIDELINES.md

```diff
@@ -142,24 +148,27 @@ Add a `make` target
 ### Some things linting will catch
 
 Unused variables, proper casing of function and variable names, undeclared global variables,
-modification of global variables, missing semi-colons. Catching unused variables is great
-because this might alert you to a typo in your code. Modifying global variables could have
-adverse affects on other code within your system, including dependencies, if the other code
-is not aware of the modifications. For these and many other reasons, we think linting is a
+modification of global variables, missing semi-colons, consistent use of const/let/var.
+Catching unused variables is great because this might alert you to a typo in your code.
+Modifying global variables could have adverse affects on other code within your system,
+including dependencies, if the other code is not aware of the modifications. Using `const`
+is preferred to `var`. If a variable must be mutable, use `let` (`var` is also allowed for
+mutable variables for cases where `let` may cause performance issues, but is restricted to
+block scope, so will act like `let`. For these and many other reasons, we think linting is a
```

Comment:
Hrmm, so this diff ain't the best, is it? I set my editor to what I determined to be the max line length in the paragraph previously (92? lol my emacs wanted it to be 70), so when I put new content in the middle, the entire paragraph got re-flowed.

One possible improvement that could be considered to lead to better diffs in your markdown (or other content files), could be to start each sentence on a new line (of course you can line-break within a long sentence also). As long as there isn't a blank line between two sentences in markdown, they should be rendered as a single paragraph (i.e. the fact that a new sentence started on a new sentence in the raw version of the file doesn't matter: it'll be re-flowed to be a unified paragraph in the rendered/rich version).

Maybe I'm thinking about this too much...we probably don't need a .mdlint.json to talk about .eslintrc.json ;)